BUILD_yarn-install.bat

```diff
@@ -0,0 +1,3 @@
+cd %~dp0

```

Comment:
A script for automating `yarn install`?

Reply:
Yes, it is. Just run by double-click.
`yarn` was been pre-installed, before run this .bat`-file on Windows 8.1 Pro.